src/Deeson/WardenThirdPartyLibraryBundle/Managers/ThirdPartyLibraryManager.php

```diff
@@ -80,7 +80,7 @@ public function onWardenSiteUpdate(SiteUpdateEvent $event) {
     $data = $event->getData();
     $site = $event->getSite();
     $libraryData = array();
-    if (isset($data->library)) {
+    if (isset($data->library)  && $data->library !== 'disabled') {
```

Comment:
I think I would change this to use to check if $data->library was an array or not rather than checking against a defined string